src/test/java/uk/gov/pay/api/json/CreateCardPaymentRequestDeserializerTest.java

```diff
@@ -432,6 +472,54 @@ public void deserialize_shouldThrowValidationException_whenDelayedCaptureIsNumer
         deserializer.deserialize(jsonFactory.createParser(json), ctx);
     }
 
+    @Test
+    public void deserialize_shouldThrowValidationException_whenMotoIsNotABoolean() throws Exception {
+        // language=JSON
+        String json = "{\n" +
+                "  \"amount\": 1337,\n" +
+                "  \"reference\": \"Some reference\",\n" +
+                "  \"description\": \"Some description\",\n" +
+                "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
+                "  \"moto\": \"true\"\n" +
+                "}";
+
+        expectedException.expect(aBadRequestExceptionWithError("P0102", "Invalid attribute value: moto. Must be true or false"));
+
+        deserializer.deserialize(jsonFactory.createParser(json), ctx);
+    }
+
+    @Test
+    public void deserialize_shouldThrowValidationException_whenMotoIsNullValue() throws Exception {
+        // language=JSON
+        String json = "{\n" +
+                "  \"amount\": 1337,\n" +
+                "  \"reference\": \"Some reference\",\n" +
+                "  \"description\": \"Some description\",\n" +
+                "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
+                "  \"moto\": null\n" +
+                "}";
+
+        expectedException.expect(aBadRequestExceptionWithError("P0102", "Invalid attribute value: moto. Must be true or false"));
+
+        deserializer.deserialize(jsonFactory.createParser(json), ctx);
+    }
+
+    @Test
+    public void deserialize_shouldThrowValidationException_whenMotoIsNumeric() throws Exception {
+        // language=JSON
+        String json = "{\n" +
+                "  \"amount\": 1337,\n" +
+                "  \"reference\": \"Some reference\",\n" +
+                "  \"description\": \"Some description\",\n" +
+                "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
+                "  \"moto\": 0\n" +
+                "}";
+
+        expectedException.expect(aBadRequestExceptionWithError("P0102", "Invalid attribute value: moto. Must be true or false"));
+
+        deserializer.deserialize(jsonFactory.createParser(json), ctx);
+    }
+
```

Comment:
For all the new tests added above, consider using Parameters, e.g. at https://github.com/alphagov/pay-connector/blob/master/src/test/java/uk/gov/pay/connector/charge/service/ChargeCancelServiceTest.java#L88-L92, in another PR.